Doc/whatsnew/3.11.rst

```diff
@@ -423,52 +423,57 @@ detected by checking its tag for a ``-32`` suffix. All releases of Python
 since 3.5 have included this in their 32-bit builds.
 
 
+.. _whatsnew311-other-lang-changes:
+
 Other Language Changes
 ======================
 
-* Starred expressions can be used in :ref:`for statements<for>`. (See
-  :issue:`46725` for more details.)
-
-* Asynchronous comprehensions are now allowed inside comprehensions in
-  asynchronous functions. Outer comprehensions implicitly become
-  asynchronous. (Contributed by Serhiy Storchaka in :issue:`33346`.)
+* Starred unpacking expressions can now be used in :keyword:`for` statements.
+  (See :issue:`46725` for more details.)
 
-* A :exc:`TypeError` is now raised instead of an :exc:`AttributeError` in
-  :meth:`contextlib.ExitStack.enter_context` and
-  :meth:`contextlib.AsyncExitStack.enter_async_context` for objects which do not
-  support the :term:`context manager` or :term:`asynchronous context manager`
-  protocols correspondingly.
-  (Contributed by Serhiy Storchaka in :issue:`44471`.)
+* Asynchronous :ref:`comprehensions <comprehensions>` are now allowed
+  inside comprehensions in :ref:`asynchronous functions <async def>`.
+  Outer comprehensions implicitly become asynchronous in this case.
+  (Contributed by Serhiy Storchaka in :issue:`33346`.)
 
 * A :exc:`TypeError` is now raised instead of an :exc:`AttributeError` in
-  :keyword:`with` and :keyword:`async with` statements for objects which do not
-  support the :term:`context manager` or :term:`asynchronous context manager`
-  protocols correspondingly.
-  (Contributed by Serhiy Storchaka in :issue:`12022`.)
-
-* Added :meth:`object.__getstate__` which provides the default
-  implementation of the ``__getstate__()`` method.  :mod:`Copying <copy>`
-  and :mod:`pickling <pickle>` instances of subclasses of builtin types
+  :keyword:`with` statements and :meth:`contextlib.ExitStack.enter_context`
+  for objects that do not support the :term:`context manager` protocol,
+  and in :keyword:`async with` statements and
+  :meth:`contextlib.AsyncExitStack.enter_async_context`
+  for objects not supporting the :term:`asynchronous context manager` protocol.
+  (Contributed by Serhiy Storchaka in :issue:`12022` and :issue:`44471`.)
+
+* Added :meth:`object.__getstate__`, which provides the default
+  implementation of the :meth:`!__getstate__` method. :mod:`copy`-ing
+  and :mod:`pickle`-ing instances of subclasses of builtin types
```

Comment:
I think I liked the previous markup better for the `mod`s.  If you want to only apply the markup to the module name, I would drop the `-` and use `copy`ing and `pickle`ing (even though  the latter is wrong).
```suggestion
  implementation of the :meth:`!__getstate__` method. :mod:`Copying <copy>`
  and :mod:`pickling <pickle>` instances of subclasses of builtin types
```
```suggestion
  implementation of the :meth:`!__getstate__` method. :mod:`copy`\ing
  and :mod:`pickle`\ing instances of subclasses of builtin types
```

Reply:
Yeah, I preferred the previous except that I didn't know any way to link the module without getting the literal formatting for the whole link, which is semantically wrong. I guess I'll go with the second here.